api/api_ethereum_test.go

```diff
@@ -230,9 +230,9 @@ func testGetHeader(t *testing.T, testAPIName string, config *params.ChainConfig)
 	header := types.CopyHeader(&types.Header{
 		ParentHash:  common.HexToHash("0xc8036293065bacdfce87debec0094a71dbbe40345b078d21dcc47adb4513f348"),
 		Rewardbase:  common.Address{},
-		TxHash:      common.HexToHash("0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"),
+		TxHash:      types.EmptyCodeHash,
```

Comment:
```suggestion
		TxHash:      types.EmptyTxRootOriginal,
```
Same for other occurences in this file.

Reply:
It seems `types.EmptyTxRootSimple` or `types.EmptyTxRootConcat` for `0xc5d2...a470`

Reply:
It seems `types.EmptyTxRootSimple` or `types.EmptyTxRootConcat` for `0xc5d2...a470`.

Reply:
@blukat29 @hyeonLewis I think that `TxHash` can be modified to `types.EmptyTxRootOriginal` , which is the expected value. The result of different hash should be recalculated accordingly.

Reply:
If it's not too important (judging from the comment "dummy header") then I think EmptyTxRootOriginal is more representative "dummy" example.

But if the test ever fails, then please go back to the ones (Simple or Concat) equivalent to the original value (0xc5d2..).

Reply:
PTAL https://github.com/kaiachain/kaia/pull/146/commits/00cd23eafa9159337e3fe6266d3f901c087c19f8